granary/test/testdata/article_with_reposts.mf2.json

```diff
@@ -17,7 +17,7 @@
         "repost-of": ["http://example.com/abc"]
       }
     }, {
-      "type": ["h-entry", "h-as-repost"],
+      "type": ["h-cite", "h-as-share"],
```

Comment:
i couldn't find meaningful mentions of either `h-as-share` or `h-as-repost` on http://indiewebcamp.com/ or http://microformats.org/wiki/microformats2 . so...got me. sgtm. :P